lessons/updated-intro-to-ar-development.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Introduction to Mobile Augmented Reality Development in Unity
+authors:
+- Jacob W. Greene
+date: 2015-01-04
+layout: lesson
+reviewers:
+- Jason Crider
+- Fred Gibbs
+- Sean Morey
+editors:
+- Fred Gibbs
+- Brandon Walsh
+difficulty: 2
+activity: presenting
+topics: [website,mapping]
+abstract: "This lesson serves as an introduction to creating mobile augmented reality applications. Augmented reality (AR) can be defined as the overlaying of digital content (images, video, text, sound, etc.) onto physical objects or locations, and it is typically experienced by looking through the camera lens of an electronic device such as a smartphone, tablet, or optical head-mounted display."
+---
+
```

Comment:
We definitely want to include a banner at the start of this lesson pointing to the retired one. 

To add the banner in here, use:

```html
<div class="alert alert-warning">
    <p>lorem ipsum etc...</p>
</div>
```

Reply:
Done in [this commit](https://github.com/programminghistorian/jekyll/pull/800/commits/d77c2bcb01ad182e5d11f9454599105bb74c6e13). I took out the <p> tags because the padding was causing some weird looking spacing, but let me know if you want it back.